lib/odometry-ng/include/odometer/OdometerDifferentialParameters.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+namespace cogip {
+
+namespace odometer {
+
+class OdometerDifferentialParameters {
+public:
+    /// @brief Construct a new Odometry Params object
+    /// @param left_wheel_diameter left wheel diameter value (mm)
+    /// @param right_wheel_diameter right wheel diameter value (mm)
+    /// @param axle_track Axle track value (mm)
+    /// @param left_polarity left wheel polarity no unit, 1 or -1
+    /// @param right_polarity riht wheel polarity no unit, 1 or -1
+    OdometerDifferentialParameters(double left_wheel_diameter, 
+    double right_wheel_diameter, double axle_track, double left_polarity, double right_polarity) : left_wheel_diameter_(left_wheel_diameter), right_wheel_diameter_(right_wheel_diameter), axle_track_(axle_track), left_polarity_(left_polarity), right_polarity_(right_polarity) {};
```

Comment:
what about `track_width` instead of `axle_track` ?

Reply:
switched to `track_width` this is a better name